src/main/java/frc/robot/commands/ExtendDeployingBar.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.commands;
+
+import frc.robot.subsystems.DeployingBar;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ExtendDeployingBar extends CommandBase {
+  private final DeployingBar m_deployingbar;
+  double setpoint;
+  /**
+   * Creates a new ExtendDeployingBar
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public ExtendDeployingBar(DeployingBar deployingbar, double setpoint) {
```

Comment:
this is minor but you should capitalize the B in deployingbar, seems to be uncapitalized in a few places